.ci/jenkins/Jenkinsfile.daily

```
@@ -106,6 +106,17 @@ pipeline {
                 }
             }
         }
+        stage('Deploy process-migration-service locally with jdk11'){
+            steps {
+                dir("${WORKSPACE}" + 'kiegroup/process-migration-service') {
```

I would say process-migration-service folder it's different than the one you are proposing...
```suggestion
                dir("${WORKSPACE}/bc/kiegroup_process_migration_service') {
```